lamindb/examples/cellxgene/_cellxgene.py

```diff
@@ -45,16 +54,19 @@ def save_cxg_defaults() -> None:
     ).save()
 
     # na, unknown
-    for model, name in zip(
+    for biorecord, name in zip(
         [
+            bt.Ethnicity,
             bt.Ethnicity,
             bt.DevelopmentalStage,
             bt.Phenotype,
             bt.CellType,
         ],
-        ["na", "unknown", "unknown", "unknown"],
+        ["na", "unknown", "unknown", "unknown", "unknown"],
```

Comment:
According to the spec, it should only be "na" but they have both. We also had both before. So this change restores the behavior.